backend/generating/ah2096_DigitalElectronics_1.tex

```diff
@@ -0,0 +1,8 @@
+\newcommand{\studentname}{Ananya Hari Narain}
+\newcommand{\studentemail}{ah2096@cam.ac.uk}
+\newcommand{\svrname}{Matthew Ireland}
+\newcommand{\svvenue}{Churchill}
+\newcommand{\svcourse}{Digital Electronics}
+\newcommand{\svnumber}{1}
+\newcommand{\svdate}{19/10/19}
+\newcommand{\svtime}{12:30}

```

Comment:
This file doesn't need to be included in the repository.

It seems as if this code is generating the headers correctly, but the LaTeX template hasn't been modified to actually use these headers. I suggest we output a file `per_supervision_headers.tex` and then `\input{per_supervision_headers.tex}` in the LaTeX template (we can include an example per_supervision_headers.tex in the template directory if we like; the current example is below).

```
% Configure these parameters.
% Name and email
\newcommand{\studentname}{Harry Potter}
\newcommand{\studentemail}{hjp1@cam.ac.uk}
\newcommand{\svvenue}{Churchill 1C}
\newcommand{\svrname}{Dr John K.~Fawcett}

% Details of supervision
\newcommand{\svcourse}{CST Part IA: Defence Against the Dark Arts}
\newcommand{\svnumber}{1}
\newcommand{\svdate}{2018-10-07}
\newcommand{\svtime}{14:00}
```